### PR TITLE
Release influxctl v2.9.5 and v2.9.6

### PR DESCRIPTION
**Pull request by powersj**
Introduces the new auth subcommand with logout and login commands. The change logs are different as only clustered will get the refresh token attempts.

- [x] Signed the [InfluxData CLA](https://www.influxdata.com/legal/cla/)
  ([if necessary](https://github.com/influxdata/docs-v2/blob/master/CONTRIBUTING.md#sign-the-influxdata-cla))
- [x] Rebased/mergeable

### CODE REVIEW

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/_index.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ influxctl [flags] [command]
 
 | Command                                                                     | Description                            |
 | :-------------------------------------------------------------------------- | :------------------------------------- |
+| [auth](/influxdb/cloud-dedicated/reference/cli/influxctl/auth/)             | Login or logout of InfluxDB v3         |
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/_index.md:35]:
```suggestion
| [auth](/influxdb/cloud-dedicated/reference/cli/influxctl/auth/)             | Log in to or log out of InfluxDB v3         |
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/_index.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+```sh
+influxctl auth [subcommand] [subcommand options] [arguments...]
+```
+
+## Subcommands
+
+| Subcommand                                                               | Description                     |
+| :----------------------------------------------------------------------- | :------------------------------ |
+| [login](/influxdb/cloud-dedicated/reference/cli/influxctl/auth/login/)   | Login with InfluxData Auth0 or InfluxDB Clustered identity provider |
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/_index.md:25]:
```suggestion
| [login](/influxdb/cloud-dedicated/reference/cli/influxctl/auth/login/)   | Log in using InfluxData Auth0 or an InfluxDB Clustered identity provider |
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/_index.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+| [logout](/influxdb/cloud-dedicated/reference/cli/influxctl/auth/logout/) | Remove local tokens             |
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/_index.md:26]:
```suggestion
| [logout](/influxdb/cloud-dedicated/reference/cli/influxctl/auth/logout/) | Remove local authorization tokens to log out of an InfluxDB cluster            |
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/_index.md (hunk 2 above)
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/_index.md:25]: > InfluxDB Clustered identity provider

Does this only apply to Clustered, or to Cloud Dedicated as well?

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/login.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+---
+title: influxctl auth login
+description: >
+  The `influxctl auth login` command logins with InfluxData Auth0 or InfluxDB
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/login.md:4]:
```suggestion
  The `influxctl auth login` command lets a user log in to an InfluxDB cluster using InfluxData Auth0 or an InfluxDB
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/login.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  Clustered identity provider
+menu:
+  influxdb_clustered:
+    parent: influxctl auth
+weight: 301
+---
+
+The `influxctl auth login` command logins with InfluxData Auth0 or InfluxDB
+Clustered identity provider
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/login.md:13]: > Clustered

Does this also apply to Cloud Ded? If so, we can use the `{{< product-name omit="Clustered" >}}` short tag. If not, I can remove it here and keep it in `/clustered...`.

Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/login.md:13]:
```suggestion
The `influxctl auth login` command lets a user log in to an InfluxDB cluster using InfluxData Auth0 or an InfluxDB Clustered identity provider.
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/logout.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
+---
+title: influxctl auth logout
+description: >
+  The `influxctl auth logout` command removes local tokens
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/logout.md:4]:
```suggestion
  The `influxctl auth logout` command logs out a user by removing local authorization tokens
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/logout.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+menu:
+  influxdb_clustered:
+    parent: influxctl auth
+weight: 301
+---
+
+The `influxctl auth logout` command removes local tokens
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/logout.md:11]:
```suggestion
The `influxctl auth logout` command logs out a user by removing local authorization tokens.
```

**Review by jstirnaman — CHANGES_REQUESTED** [review]

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/_index.md (hunk 2 above)
Comment (powersj) [reply content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/_index.md:25]: Both, dedicated is the InfluxData Auth0

File: content/influxdb/clustered/reference/cli/influxctl/auth/login.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+  Clustered identity provider
+menu:
+  influxdb_clustered:
+    parent: influxctl auth
+weight: 301
+---
+
+The `influxctl auth login` command logins with InfluxData Auth0 or InfluxDB
+Clustered identity provider
+
```
Comment (jstirnaman) [inline content/influxdb/clustered/reference/cli/influxctl/auth/login.md:14]:
```suggestion
The `influxctl auth login` command lets a user log in to an
{{< product-name omit="Clustered" >}} cluster using the cluster's configured
identity provider.  
```

File: content/influxdb/clustered/reference/cli/influxctl/auth/login.md [hunk 9]
```diff
@@ -0,0 +1,29 @@
+---
+title: influxctl auth login
+description: >
+  The `influxctl auth login` command logins with InfluxData Auth0 or InfluxDB
+  Clustered identity provider
+menu:
```
Comment (jstirnaman) [inline content/influxdb/clustered/reference/cli/influxctl/auth/login.md:6]:
```suggestion
  The `influxctl auth login` command lets a user log in to an InfluxDB cluster using
  the cluster's configured identity provider.
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/login.md [hunk 10]
```diff
@@ -0,0 +1,29 @@
+---
+title: influxctl auth login
+description: >
+  The `influxctl auth login` command logins with InfluxData Auth0 or InfluxDB
+  Clustered identity provider
+menu:
+  influxdb_clustered:
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/login.md:7]:
```suggestion
  influxdb_cloud_dedicated:
```

**Comment by powersj** [comment]
@jstirnaman,

I've pushed an update to:

1. Release v2.9.6, small bug fix
2. Update change log
3. Update the messages in auth pages to be more specific to clustered or dedicated

Hope those work better!

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/_index.md [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+
+```sh
+influxctl auth [subcommand] [subcommand options] [arguments...]
+```
+
+## Subcommands
+
+| Subcommand                                                               | Description                     |
+| :----------------------------------------------------------------------- | :------------------------------ |
+| [login](/influxdb/cloud-dedicated/reference/cli/influxctl/auth/login/)   | Log in using InfluxData Auth0   |
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/_index.md:25]:
```suggestion
| [login](/influxdb/cloud-dedicated/reference/cli/influxctl/auth/login/)   | Log in to an InfluxDB cluster using InfluxData Auth0 |
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/login.md [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+description: >
+  The `influxctl auth login` command lets a user log in to an InfluxDB cluster using
+  {{< product-name omit="Clustered" >}} identity provider
+menu:
+  influxdb_cloud_dedicated:
+    parent: influxctl auth
+weight: 301
+---
+
+The `influxctl auth login` command lets a user log in to an {{< product-name >}}
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/login.md:12]:
```suggestion
The `influxctl auth login` command lets a user log in to an {{< product-name omit="Clustered" >}}
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/_index.md [hunk 13]
```diff
@@ -0,0 +1,33 @@
...
+| [logout](/influxdb/cloud-dedicated/reference/cli/influxctl/auth/logout/) | Remove local authorization tokens to log out of an InfluxDB cluster            |
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/_index.md:26]:
```suggestion
| [logout](/influxdb/cloud-dedicated/reference/cli/influxctl/auth/logout/) | Log out of an InfluxDB cluster; remove local authorization tokens |
```

File: content/influxdb/clustered/reference/cli/influxctl/auth/logout.md [hunk 14]
```diff
@@ -0,0 +1,27 @@
...
+description: >
+  The `influxctl auth logout` command removes local tokens
+menu:
+  influxdb_clustered:
+    parent: influxctl auth
+weight: 301
+---
+
+The `influxctl auth logout` command removes local tokens
+
```
Comment (jstirnaman) [inline content/influxdb/clustered/reference/cli/influxctl/auth/logout.md:12]:
```suggestion
The `influxctl auth logout` command lets a user logout of an {{< product-name omit="Clustered" >}}
cluster; removes the user's local authorization tokens.
```

File: content/influxdb/clustered/reference/cli/influxctl/auth/_index.md [hunk 15]
```diff
@@ -0,0 +1,33 @@
...
+
+```sh
+influxctl auth [subcommand] [subcommand options] [arguments...]
+```
+
+## Subcommands
+
+| Subcommand                                                         | Description                     |
+| :----------------------------------------------------------------- | :------------------------------ |
+| [login](/influxdb/clustered/reference/cli/influxctl/auth/login/)   | Login with InfluxDB Clustered identity provider |
```
Comment (jstirnaman) [inline content/influxdb/clustered/reference/cli/influxctl/auth/_index.md:25]:
```suggestion
| [login](/influxdb/clustered/reference/cli/influxctl/auth/login/)   | Log in to an InfluxDB cluster using the cluster's identity provider |
```

File: content/influxdb/clustered/reference/cli/influxctl/auth/login.md [hunk 16]
```diff
@@ -0,0 +1,31 @@
...
+  influxdb_clustered:
+    parent: influxctl auth
+weight: 301
+---
+
+The `influxctl auth login` command logins with InfluxData Auth0 or InfluxDB
+Clustered identity provider
+The `influxctl auth login` command lets a user log in to an
+{{< product-name omit="Clustered" >}} cluster using the cluster's configured
+identity provider.
```
Comment (jstirnaman) [inline content/influxdb/clustered/reference/cli/influxctl/auth/login.md:16]:
```suggestion
The `influxctl auth login` command lets a user log in to an {{< product-name omit="Clustered" >}}
cluster using the cluster's configured identity provider. 
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/logout.md [hunk 17]
```diff
@@ -0,0 +1,27 @@
+---
+title: influxctl auth logout
+description: >
+  The `influxctl auth logout` command logs out a user by removing local authorization tokens
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/logout.md:4]:
```suggestion
  The `influxctl auth logout` command lets a user logout of an InfluxDB 
  cluster; removes the user's local authorization tokens.  
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/logout.md [hunk 18]
```diff
@@ -0,0 +1,27 @@
...
+menu:
+  influxdb_cloud_dedicated:
+    parent: influxctl auth
+weight: 301
+---
+
+The `influxctl auth logout` command logs out a user by removing local authorization tokens.
```
Comment (jstirnaman) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/logout.md:11]:
```suggestion
The `influxctl auth logout` command lets a user logout of an {{< product-name omit="Clustered" >}} 
cluster; removes the user's local authorization tokens.
```

**Review by jstirnaman — CHANGES_REQUESTED** [review]
Sorry this is so messy - I think I lost/botched some of my earlier suggestions.

File: data/products.yml [hunk 19]
```diff
@@ -55,7 +55,7 @@ influxdb_cloud_dedicated:
   list_order: 3
   latest: cloud-dedicated
   link: "https://www.influxdata.com/contact-sales-form/"
-  latest_cli: 2.9.4
```
[the remark this answers is not in the document]
Comment (powersj) [reply data/products.yml:58]: I have assumed it inherits it as well as my previous PRs have only changed this one line.

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/logout.md [hunk 20]
```diff
@@ -0,0 +1,27 @@
...
+description: >
+  The `influxctl auth logout` command logs out a user by removing local authorization tokens
+menu:
+  influxdb_clustered:
+    parent: influxctl auth
+weight: 301
+---
+
+The `influxctl auth logout` command logs out a user by removing local authorization tokens.
+
```
[the remark this answers is not in the document]
Comment (powersj) [reply content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/logout.md:14]: I accepted another suggestion from you on this same line, so I think we can reject this one?

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/_index.md (hunk 2 above)
Comment (powersj) [reply content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/_index.md]: Right

File: content/influxdb/clustered/reference/cli/influxctl/auth/_index.md [hunk 21]
```diff
@@ -0,0 +1,33 @@
+---
+title: influxctl auth
+description: >
+  The `influxctl auth` command and its subcommands provide the ability to
+  login and logout.
```
[the remark this answers is not in the document]
Comment (jstirnaman) [reply content/influxdb/clustered/reference/cli/influxctl/auth/_index.md:5]: I'm fine with saying it differently, but verbs are "log in" and "log out"; nouns are " login" and "logout":
- `my login`
- `to log in`

**Review by jstirnaman — CHANGES_REQUESTED** [review]

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/login.md [hunk 22]
```diff
@@ -0,0 +1,29 @@
+---
+title: influxctl auth login
+description: >
+  The `influxctl auth login` command lets a user log in to an InfluxDB cluster using
+  {{< product-name omit="Clustered" >}} identity provider
```
Comment (sanderson) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/login.md:5]: Unfortunately, Hugo shortcodes don't work in page descriptions.

```suggestion
  the InfluxDB Cloud Dedicated identity provider.
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/logout.md [hunk 23]
```diff
@@ -0,0 +1,29 @@
+---
+title: influxctl auth logout
+description: >
+  The `influxctl auth logout` command lets a user logout of an InfluxDB 
+  cluster; removes the user's local authorization tokens.
```
Comment (sanderson) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/logout.md:5]:
```suggestion
  The `influxctl auth logout` command lets a user log out of an InfluxDB 
  cluster and removes the user's local authorization tokens.  
```

File: content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/logout.md [hunk 24]
```diff
@@ -0,0 +1,29 @@
...
+  The `influxctl auth logout` command lets a user logout of an InfluxDB 
+  cluster; removes the user's local authorization tokens.  
+menu:
+  influxdb_cloud_dedicated:
+    parent: influxctl auth
+weight: 301
+---
+
+The `influxctl auth logout` command lets a user logout of an {{< product-name omit="Clustered" >}} 
+cluster; removes the user's local authorization tokens.
```
Comment (sanderson) [inline content/influxdb/cloud-dedicated/reference/cli/influxctl/auth/logout.md:13]:
```suggestion
The `influxctl auth logout` command lets a user log out of an {{< product-name omit="Clustered" >}} 
cluster and removes the user's local authorization tokens.
```

File: content/influxdb/clustered/reference/cli/influxctl/auth/logout.md [hunk 25]
```diff
@@ -0,0 +1,29 @@
+---
+title: influxctl auth logout
+description: >
+  The `influxctl auth logout` command lets a user logout of an InfluxDB 
+  cluster; removes the user's local authorization tokens.
```
Comment (sanderson) [inline content/influxdb/clustered/reference/cli/influxctl/auth/logout.md:5]:
```suggestion
  The `influxctl auth logout` command lets a user log out of an InfluxDB 
  cluster and removes the user's local authorization tokens.
```

File: content/influxdb/clustered/reference/cli/influxctl/auth/logout.md [hunk 26]
```diff
@@ -0,0 +1,29 @@
...
+menu:
+  influxdb_clustered:
+    parent: influxctl auth
+weight: 301
+---
+
+The `influxctl auth logout` command lets a user logout of an {{< product-name omit="Clustered" >}}
+cluster; removes the user's local authorization tokens.
```
Comment (sanderson) [inline content/influxdb/clustered/reference/cli/influxctl/auth/logout.md:13]:
```suggestion
The `influxctl auth logout` command lets a user log out of an {{< product-name omit="Clustered" >}}
cluster and removes the user's local authorization tokens.
```

File: data/products.yml (hunk 19 above)
[the remark this answers is not in the document]
Comment (sanderson) [reply data/products.yml:58]: Yes, it does inherit it since they use the same CLI.

**Review by sanderson — APPROVED** [review]
Thanks for this @powersj! Just some minor suggestions and I think it's gtg.

**Review by sanderson — APPROVED** [review]
🚀

**Comment by powersj** [comment]
@srebhan it looks like something changed with the CLA bot, as now it thinks I haven't signed.

@jstirnaman need you to +1 or dismiss your review as I couldn't merge since you requested changes

**Review by jstirnaman — APPROVED** [review]

**Pull request merged**